### PR TITLE
modifying build phase because of FindCUDA.cmake behaviour

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

File: var/spack/repos/builtin/packages/brayns/package.py [hunk 1]
```diff
@@ -68,3 +69,9 @@ def cmake_args(self):
     def check(self):
         with working_dir(self.build_directory):
             ninja('tests')
+
+    def build(self, spec, prefix):
+        chdir('spack-build')
+        if '+optix' in self.spec:
+            ninja('braynsOptixEngine')
```
Comment (dnachbaur) [inline var/spack/repos/builtin/packages/brayns/package.py]: you could also 'ninja -C spack-build braynsOptixEngine', or `with working_dir(self.build_directory)` like above in check()

Comment (matz-e) [reply var/spack/repos/builtin/packages/brayns/package.py]: I think the latter would be best.

Comment (ppodhajski) [reply var/spack/repos/builtin/packages/brayns/package.py]: done

**Review by matz-e — APPROVED** [review]
I approve!

**Comment by pramodk** [comment]
Just as a reminder: once we merge PR, we should delete branch.

**Pull request merged**